GoogleDataTransport/GDTLibrary/GDTAssert.m

```diff
@@ -14,9 +14,9 @@
  * limitations under the License.
  */
 
-#import "GDTLibrary/Private/GDTAssert.h"
+#import "GDTLibrary/Public/GDTAssert.h"
```

Comment:
Why Public?

Reply:
Because it needs to be visible to GDTCCT and I've been trying to avoid module imports in implementation files.